examples/nimble_heart_rate_sensor/README.md

```diff
@@ -5,6 +5,9 @@ notifications by implementing a mock-up BLE heart rate sensor. Once flashed on
 a node, you can connect to it using any phone or device capable of using such a
 heart rate sensor, e.g. Nordics `nRF Toolbox` for Android
 (https://play.google.com/store/apps/details?id=no.nordicsemi.android.nrftoolbox).
+With browsers supporting the experimental [Web Bluetooth API](https://webbluetoothcg.github.io/web-bluetooth/)
+(currently only the Chromium / Chrome family), this can also be accessed from a
```

Comment:
It's probably a good idea to add a note about this currently not working on the desktop, so someone who tries that out won't be pulling their hair to no end, wondering why it's not working for them.